import-export-cli/integration/apim/apim.go

```diff
@@ -2039,6 +2050,234 @@ func (instance *Client) registerClient(username string, password string) dcrResp
 	return jsonResp
 }
 
+// AddThrottlePolicy : Add new Throttle Policy of different policy types to APIM
+func (instance *Client) AddThrottlePolicy(t *testing.T, policy interface{}, username, password, policyType string, doClean bool) map[string]interface{} {
+	var throttlePolicyResponse map[string]interface{}
+
+	throttlePolicyURL := instance.adminRestURL + "/throttling/policies/" + policyType
+
+	data, err := json.Marshal(policy)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	request := base.CreatePost(throttlePolicyURL, bytes.NewBuffer(data))
+
+	base.SetDefaultRestAPIHeaders(instance.accessToken, request)
+
+	base.LogRequest("apim.AddThrottlePolicy()", request)
+
+	response := base.SendHTTPRequest(request)
+
+	defer response.Body.Close()
+
+	base.ValidateAndLogResponse("apim.AddThrottlePolicy()", response, 201)
+
+	json.NewDecoder(response.Body).Decode(&throttlePolicyResponse)
+	policyId := fmt.Sprintf("%v", throttlePolicyResponse["policyId"])
+	if doClean {
+		t.Cleanup(func() {
+			instance.Login(username, password)
+			instance.DeleteThrottlePolicy(policyId, policyType)
+		})
+	}
+	return throttlePolicyResponse
+}
+
+// DeleteThrottlePolicy : Deletes Throttling Policy from APIM using UUID
+func (instance *Client) DeleteThrottlePolicy(policyID, policyType string) {
+
```

Comment:
Remove the empty lines added at the beginning of function body. Fix at other places as well

Reply:
fixed with 0fd0d5442f47bfc4418415b38bf56c03bfc1e655